holoviews/plotting/bokeh/callbacks.py

```diff
@@ -272,54 +340,109 @@ def set_customjs(self, handle, references):
                                                 debounce=self.debounce)
 
         attributes = attributes_js(self.attributes, references)
-        code = 'var data = {};\n' + attributes + self.code + self_callback
-
-        # Merge callbacks if another callback has already been attached
-        # otherwise set it
-        if id(handle.callback) in self._callbacks:
-            cb = self._callbacks[id(handle.callback)]
+        conditions = ["%s" % cond for cond in self.skip]
+        conditional = ''
+        if conditions:
+            conditional = 'if (%s) { return };\n' % (' || '.join(conditions))
+        data = "var data = {};\n"
+        code = conditional + data + attributes + self.code + self_callback
+
+        js_callback = CustomJS(args=references, code=code)
```

Comment:
I would consider factoring out a method that returns a ``CustomJS`` so all the Javascript is kept separate (probably including the call to ``attributes_js``). The method could be called something like ``get_customjs``...